### PR TITLE
Addition of drink questions and drink continuation.

### PR DESCRIPTION
**Pull request by jayzamazing**
This covers the bartender asking what kind of drink the person wants along with asking the user if they want another drink and generating a random drink.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.answer-choices label {
+    margin-left: 0 !important;
```
Comment (aric87) [inline css/style.css:11]: Consider whether you _need_ !important here, or could you change the specificity. !important had its place, but often leads to a heap of them

File: index.html [hunk 2]
```diff
@@ -1,14 +1,82 @@
...
+        </div>
+    </header>
+    <section class="question-section container">
+    </section>
+    <script src="bower_components/jquery/dist/jquery.min.js"></script>
+    <script src="bower_components/bootstrap/dist/js/bootstrap.min.js"></script>
+    <script src="bower_components/handlebars/handlebars.js"></script>
+    <script src="js/app.js"></script>
+    <script id="question-template" type="text/x-handlebars-template">
+        <form action="" id="question-form">
```
Comment (aric87) [inline index.html]: If you don't need an action, remove the attribute.

File: index.html [hunk 3]
```diff
@@ -1,14 +1,82 @@
...
+            </div>
+        </form>
+    </script>
+    <script id="another-drink-template" type="text/x-handlebars-template">
+        <div class="row">
+            <div class="currentDrink col-xs-12 col-sm-12 col-md-8 col-md-offset-2 col-lg-8 col-lg-offset-2">
+                <h2>{{drink}}</h2>
+            </div>
+        </div>
+        <form action="" id="another-drink-form">
```
Comment (aric87) [inline index.html]: Ditto

File: index.html [hunk 4]
```diff
@@ -1,14 +1,82 @@
...
+            <div class="row">
+                <div class="newDrink col-xs-12 col-sm-12 col-md-8 col-md-offset-2 col-lg-8 col-lg-offset-2">
+                    <h2>Would ye be having another drink?</h2>
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-xs-12 col-sm-12 col-md-8 col-md-offset-2 col-lg-8 col-lg-offset-2">
+                    <div class="col-centered new-drink-submit col-lg-2">
+                        <input type="submit" name="submit" class="btn btn-default btn-primary col-lg-12" value="YARRR!">
```
Comment (aric87) [inline index.html:75]: This needs a sound effect! :)

**Comment by jayzamazing** [comment]
!important is being used to override bootstrap implementation. Added sound effect and background image, although I am not that great when it comes to visual stuff. Removed action.

**Pull request merged**